`make test` fails on Fedora24

example `make test` output on F24:

```
Successfully built 30ae53632fda
2016-07-05 13:36:43,743 root         INFO     Running version 0.4.4, Docker ee81b72/1.10.3, API 1.22...
2016-07-05 13:36:43,743 root         INFO     Squashing image 'openshift/nodejs-010-centos7-candidate'...
2016-07-05 13:36:43,783 root         INFO     Old image has 13 layers
2016-07-05 13:36:43,783 root         INFO     Attempting to squash from layer sha256:40d63144482c811b217579cce280299b2c129ac60e904b2ef594131b51d06aa6...
2016-07-05 13:36:43,783 root         INFO     Checking if squashing is necessary...
2016-07-05 13:36:43,783 root         INFO     We have 12 layers to squash
2016-07-05 13:36:43,785 root         INFO     Saving image sha256:30ae53632fdada7c8050da88deca7e8552ce6d4e76b6db9ebb552a88c549f2af to /tmp/docker-squash-fMonNb/image.tar file...
2016-07-05 13:37:00,389 root         INFO     Image saved!
2016-07-05 13:37:00,413 root         INFO     Unpacking /tmp/docker-squash-fMonNb/image.tar tar file to /tmp/docker-squash-fMonNb/old directory
2016-07-05 13:37:00,749 root         INFO     Archive unpacked!
2016-07-05 13:37:00,750 root         INFO     New layer ID for squashed content will be: c3ba05ecea7c4d24682f663e8074e02ba916b481fa8b423eba88815bda581c63
2016-07-05 13:37:00,750 root         INFO     Starting squashing...
2016-07-05 13:37:00,750 root         INFO     Squashing layer 'sha256:30ae53632fdada7c8050da88deca7e8552ce6d4e76b6db9ebb552a88c549f2af'...
Traceback (most recent call last):
  File "/home/ryanj/.local/bin/docker-scripts", line 9, in <module>
    load_entry_point('docker-scripts==0.4.4', 'console_scripts', 'docker-scripts')()
  File "build/bdist.linux-x86_64/egg/docker_scripts/cli.py", line 96, in run
  File "build/bdist.linux-x86_64/egg/docker_scripts/cli.py", line 88, in run
  File "build/bdist.linux-x86_64/egg/docker_scripts/cli.py", line 31, in run_squash
  File "build/bdist.linux-x86_64/egg/docker_scripts/squash.py", line 497, in run
  File "build/bdist.linux-x86_64/egg/docker_scripts/squash.py", line 337, in _squash_layers
  File "/usr/lib64/python2.7/tarfile.py", line 1673, in open
    return func(name, "r", fileobj, **kwargs)
  File "/usr/lib64/python2.7/tarfile.py", line 1738, in gzopen
    fileobj = gzip.GzipFile(name, mode, compresslevel, fileobj)
  File "/usr/lib64/python2.7/gzip.py", line 94, in __init__
    fileobj = self.myfileobj = __builtin__.open(filename, mode or 'rb')
IOError: [Errno 2] No such file or directory: u'/tmp/docker-squash-fMonNb/old/sha256:30ae53632fdada7c8050da88deca7e8552ce6d4e76b6db9ebb552a88c549f2af/layer.tar'
hack/common.mk:25: recipe for target 'test' failed
```

--- a/hack/build.sh
+++ b/hack/build.sh
@@ -44,13 +44,13 @@ function docker_build_with_version {
 }
 
 # Install the docker squashing tool[1] and squash the result image
-# [1] https://github.com/goldmann/docker-scripts
+# [1] https://github.com/goldmann/docker-squash
 function squash {
   # FIXME: We have to use the exact versions here to avoid Docker client
   #        compatibility issues
-  easy_install -q --user docker_py==1.6.0 docker-scripts==0.4.4
+  easy_install -q --user docker_py==1.6.0 docker-squash==1.0.0rc6
   base=$(awk '/^FROM/{print $2}' $1)
-  ${HOME}/.local/bin/docker-scripts squash -f $base ${IMAGE_NAME}
+  ${HOME}/.local/bin/docker-squash -f $base ${IMAGE_NAME}
 }
 
 # Versions are stored in subdirectories. You can specify VERSION variable

